rfc_template.md

```diff
@@ -0,0 +1,18 @@
+# 0001: RFC_NAME
+
+**Updated:** January 01, 2018  
```

Comment:
Since files are committed markdown files with a git history, it might not be necessary to use a "Updated" column. Maybe it's enough to keep dates for state changes?

Reply:
Yeah, could be fine! I'm down to remove this.